rpmio/expression.c

```diff
@@ -438,6 +438,10 @@ static Value doMultiplyDivide(ParseState state)
     if (valueIsInteger(v1)) {
       int i1 = v1->data.i, i2 = v2->data.i;
 
+      if ((i2 == 0) && (TOK_DIVIDE)) {
```

Comment:
Is `(TOK_DIVIDE)` supposed to be `(op == TOK_DIVIDE)`?

Reply:
Ugh, yeah:

```
$ ./rpm --eval "%{expr:1*0}"
error: division by zero: 1*0
```


Thanks for spotting, I admit I barely looked at the implementation, this is originally @pavlinamv 's patch in PR #833.